src/Reference/ReferencesList.tsx

```diff
@@ -0,0 +1,70 @@
+import './ReferencesList.css'
+
+import {
+    forwardRef, useEffect, useImperativeHandle,
+    useState,
+} from 'react'
+
+export default forwardRef((props: { items: Array<string>, command: ({ id }: { id: string }) => any }, ref) => {
```

Comment:
use `=> void` instead of `=> any` if you don't care about the return type of a callback.